gbn/gbn_conn.go

```diff
@@ -89,63 +84,62 @@ func newGoBackNConn(ctx context.Context, cfg *config,
 	prefix := fmt.Sprintf("(%s)", loggerPrefix)
 	plog := build.NewPrefixLog(prefix, log)
 
+	timeoutManager := NewTimeOutManager(true)
```

Comment:
server = true for both?

Reply:
Addressed to ensure the intermediate commits also has correct `is_server` values!